src/features/world/ui/factions/JoinFaction.tsx

```diff
@@ -0,0 +1,166 @@
+import { useAppTranslation } from "lib/i18n/useAppTranslations";
+
+import sflIcon from "assets/icons/sfl.webp";
+import { Context } from "features/game/GameProvider";
+import { MachineState } from "features/game/lib/gameMachine";
+import { useSelector } from "@xstate/react";
+import { capitalize } from "lib/utils/capitalize";
+import { FactionEmblem, FactionName } from "features/game/types/game";
+import { FACTION_BANNERS } from "features/game/events/landExpansion/pledgeFaction";
+import {
+  EMBLEM_QTY,
+  SFL_COST,
+} from "features/game/events/landExpansion/joinFaction";
+import { ClaimReward } from "features/game/expansion/components/ClaimReward";
+import { useSound } from "lib/utils/hooks/useSound";
+
+const FACTION_EMBLEM: Record<FactionName, FactionEmblem> = {
+  sunflorians: "Sunflorian Emblem",
+  bumpkins: "Bumpkin Emblem",
+  goblins: "Goblin Emblem",
+  nightshades: "Nightshade Emblem",
+};
+
+const RECRUITER_VOICE: Record<FactionName, string> = {
+  goblins: "goblin_recruiter",
+  bumpkins: "bumpkin_recruiter",
+  sunflorians: "sunflorian_recruiter",
+  nightshades: "nightshade_recruiter",
+};
+
+interface Props {
+  faction: FactionName;
+  onClose: () => void;
+}
+
+const _joinedFaction = (state: MachineState) => state.context.state.faction;
+
+export const JoinFaction: React.FC<Props> = ({ faction, onClose }) => {
+  const { gameService } = useContext(Context);
+  const { t } = useAppTranslation();
+  const [showConfirm, setShowConfirm] = useState(false);
+  const [confirmed, setConfirmed] = useState(false);
+
+  const joinedFaction = useSelector(gameService, _joinedFaction);
+  const recruiterVoice = useSound(RECRUITER_VOICE[faction] as any);
+
+  const sameFaction = joinedFaction && joinedFaction.name === faction;
+
+  useEffect(() => {
+    if (sameFaction) {
+      recruiterVoice.play();
+    }
+  }, [joinedFaction]);
+
+  const handlePledge = () => {
+    gameService.send("faction.joined", { faction });
+    recruiterVoice.play();
+  };
+
+  // If joined a different faction, show a message that they can't change
+  if (joinedFaction && joinedFaction.name !== faction) {
+    return (
+      <>
+        <div className="flex flex-col p-2 pt-1 space-y-2">
+          <div className="flex justify-between">
+            <Label type="default">{capitalize(faction)}</Label>
+            <Label type="danger">{`No Access`}</Label>
+          </div>
+          <span className="text-xs sm:text-sm">
+            {t("faction.restrited.area", { faction: capitalize(faction) })}
+          </span>
+        </div>
+      </>
+    );
+  }
+
+  return (
+    <>
+      {!showConfirm && !joinedFaction && (
+        <>
+          <div className="flex flex-col px-2 py-1 space-y-2">
+            <div className="flex justify-between">
+              <Label type="default">{capitalize(faction)}</Label>
+              <Label type="danger">{`No Access`}</Label>
+            </div>
+            <span className="text-xs sm:text-sm">
```

Comment:
Are we able to use the InlineDialogue for the intro, that way it feels more like a conversation 👍